views/Tenant.rb

```diff
@@ -0,0 +1,4 @@
+class Tenant
```

Comment:
A couple things...

1. Don't forge to close you class with `end`!
2. You should instantiate your instance variables inside of an initialize method. You also need to pass arguments into it. So something like this...

```rb
class Tenant
  def initialize( name, age, gender )
    @name = name
    @age = age
    @gender = gender
  end
end
```

(Keep in mind, once you make your classes inherit from ActiveRecord, they won't need initialize methods anymore!)